web/src/test/java/org/springframework/security/web/util/matcher/MediaTypeRequestMatcherTests.java

```diff
@@ -189,6 +319,15 @@ public void useEqualsWithCustomMediaType() throws HttpMediaTypeNotAcceptableExce
 		assertThat(matcher.matches(request)).isTrue();
 	}
 
+	@Test
+	public void useEqualsWhenTrueThenCustomMediaTypeIsMatched() throws HttpMediaTypeNotAcceptableException {
```

Comment:
A number of these tests don't actually throw an `HttpMediaTypeNotAcceptableException`. Will you please remove the `throws` from the signature where it is unnecessary? This way, the expectations of the test are clearer.

Reply:
ok